controller/src/server/functions.ts

```diff
@@ -1,9 +1,16 @@
 /* eslint-disable eslint-comments/disable-enable-pair */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable node/no-unsupported-features/es-syntax */
 /* eslint-disable @typescript-eslint/no-unsafe-call */
 /* eslint-disable no-restricted-syntax */
```

Comment:
It starts to be a lot of typescrript warnings and errors being ignored. I think we should aim to not do that 😊

Reply:
I have addressed these in the latest commit